.circleci/config.yml

```diff
@@ -191,22 +191,22 @@ jobs:
       - capture_test_results
       - capture_test_logs
 
-  acceptanceTestsQuorum:
-    parallelism: 1
-    executor: xl_machine_executor
-    steps:
-      - prepare
-      - attach_workspace:
-          at: ~/project
-      - run:
-          name: Quorum Acceptance Tests
-          no_output_timeout: 40m
-          command: ./gradlew --no-daemon acceptanceTestsQuorum
-      - store_artifacts:
-          path: build/quorum-at
-          destination: quorum-at-artifacts
-      - store_test_results:
-          path: build/quorum-at/openjdk-latest/reports/xml-report
+#  acceptanceTestsQuorum:
+#    parallelism: 1
+#    executor: xl_machine_executor
+#    steps:
+#      - prepare
+#      - attach_workspace:
+#          at: ~/project
+#      - run:
+#          name: Quorum Acceptance Tests
+#          no_output_timeout: 40m
+#          command: ./gradlew --no-daemon acceptanceTestsQuorum
```

Comment:
If you change just this line this will ignore if the tests fail
`          command: ./gradlew --no-daemon acceptanceTestsQuorum || true # ignore result of job for now, until all issues are fixed`

Reply:
That is a very good idea! Much better that what I did :-)